edge-util/src/Microsoft.Azure.Devices.Edge.Util/CertificateHelper.cs

```diff
@@ -258,6 +265,33 @@ public static IList<string> ParsePemCerts(string pemCerts)
                 .ToList(); // Re-add the certificate end-marker which was removed by split
         }
 
+        public static IEnumerable<X509Certificate2> ParseTrustedBundleFromFile(string trustBundleFilePath)
+        {
+            string certs;
+
+            if (string.IsNullOrWhiteSpace(trustBundleFilePath) || !File.Exists(trustBundleFilePath))
+            {
+                throw new ArgumentException($"'{trustBundleFilePath}' is not a path to a trust bundle certificates file");
+            }
+
+            using (var sr = new StreamReader(trustBundleFilePath))
+            {
+                certs = sr.ReadToEnd();
+            }
+
+            return ParseTrustedBundleCerts(certs);
+        }
+
+        internal static IEnumerable<X509Certificate2> ParseTrustBundleResponse(TrustBundleResponse response)
+        {
+            return ParseTrustedBundleCerts(response.Certificate);
```

Comment:
Do we need to check response for null?

Reply:
Nit: Change to expression bodied syntax..

Reply:
Yes I can add this. Makes sense.

Reply:
Added to https://github.com/Azure/iotedge/pull/562